dex-load/src/main/scala/com/wavesplatform/dex/load/request/Request.scala

```diff
@@ -49,6 +49,7 @@ case class Request(
     pw.println(httpType match {
       case RequestType.POST => mkPost(jsonBody, path, tag, stringBody)
       case RequestType.GET => mkGet(path, tag, headers)
+      case _ => throw new IllegalArgumentException(s"Unexpected request type")
```

Comment:
Let's write the httpType for easier debugging:
```scala
case _ => throw new IllegalArgumentException(s"Unexpected request type: $httpType")
```

Reply:
fixed